itou/gps/migrations/0001_initial.py

```diff
@@ -48,7 +48,7 @@ class Migration(migrations.Migration):
                     "created_at",
                     models.DateTimeField(default=django.utils.timezone.now, verbose_name="date de création"),
                 ),
-                ("ended_at", models.DateTimeField(null=True, verbose_name="date de désactivation")),
+                ("ended_at", models.DateTimeField(null=True, blank=True, verbose_name="date de fin de suivi")),
```

Comment:
nit: django ordonne les args alphabétiquement

```suggestion
                ("ended_at", models.DateTimeField(blank=True, null=True, verbose_name="date de fin de suivi")),
```